### PR TITLE
fix(databricks-jdbc): databricks multiple downloads of jar

### PR DESCRIPTION
**Pull request by frolovdev**
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Issue Reference this PR resolves**

[For example #12]

**Description of Changes Made (if issue reference is not provided)**

[Description goes here]

### CODE REVIEW

File: packages/cubejs-backend-shared/src/env.ts [hunk 1]
```diff
@@ -204,7 +204,7 @@ const variables: Record<string, (...args: any) => any> = {
   databrickToken: () => get('CUBEJS_DB_DATABRICKS_TOKEN')
     .asString(),
   databrickAcceptPolicy: () => get('CUBEJS_DB_DATABRICKS_ACCEPT_POLICY')
-    .asString(),
+    .asBoolStrict(),
```
Comment (frolovdev) [inline packages/cubejs-backend-shared/src/env.ts:207]: 📓 looks like a typo

File: packages/cubejs-databricks-jdbc-driver/src/DatabricksDriver.ts [hunk 2]
```diff
@@ -64,18 +64,13 @@ const DatabricksToGenericType: Record<string, string> = {
   'decimal(10,0)': 'bigint',
 };
 
-const jdbcDriverResolver: Promise<string> | null = null;
-
 async function resolveJDBCDriver(): Promise<string> {
-  if (jdbcDriverResolver) {
-    return jdbcDriverResolver;
-  }
```
Comment (frolovdev) [inline packages/cubejs-databricks-jdbc-driver/src/DatabricksDriver.ts:72]: 📓 dead code

File: packages/cubejs-databricks-jdbc-driver/src/installer.ts [hunk 3]
```diff
@@ -16,33 +15,8 @@ function acceptedByEnv() {
   return acceptStatus;
 }
 
-async function cliAcceptVerify() {
```
Comment (frolovdev) [inline packages/cubejs-databricks-jdbc-driver/src/installer.ts:19]: 📓 now can only be configured with env variable

File: packages/cubejs-databricks-jdbc-driver/src/post-install.ts [hunk 4]
```diff
@@ -2,11 +2,15 @@ import 'source-map-support/register';
...
 import { displayCLIError } from '@cubejs-backend/shared';
 
+import fs from 'fs';
+import path from 'path';
 import { downloadJDBCDriver } from './installer';
 
 (async () => {
   try {
-    await downloadJDBCDriver(true);
+    if (!fs.existsSync(path.join(__dirname, '..', 'download', 'SparkJDBC42.jar'))) {
```
Comment (frolovdev) [inline packages/cubejs-databricks-jdbc-driver/src/post-install.ts:11]: 📓 don't reload on every yarn install

**Comment by VinThin** [comment]
@frolovdev - we are trying to use this driver but simple not able to work, my js skills are super rusty this is my issue

File: packages/cubejs-databricks-jdbc-driver/src/post-install.ts (hunk 4 above)
Comment (buntarb) [reply packages/cubejs-databricks-jdbc-driver/src/post-install.ts:11]: This block could be called more than onece, which could be a cause for a second download. You should check not only the file existing, but also a promise of the download function. Isn't it?

Comment (frolovdev) [reply packages/cubejs-databricks-jdbc-driver/src/post-install.ts:11]: Didn't get the point, in which case we can call it more than once? Two parallel yarn installs looks like a redundant case or do you mean smth else?

**Review by buntarb — APPROVED** [review]

**Pull request merged**